src/docfx/lib/markdown/MarkdigUtility.cs

```diff
@@ -153,6 +154,69 @@ public static string ToPlainText(this MarkdownObject containerBlock)
             return writer.ToString();
         }
 
+        public static bool IsVisible(this MarkdownObject markdownObject)
+        {
+            var visiable = false;
+            Visit(markdownObject, node =>
+            {
+                if (node is HtmlBlock htmlBlock)
```

Comment:
nit: consider `switch case`?
```suggestion
                if (node is HtmlBlock htmlBlock)
```